remotespark/utils/eventshandler.py

```diff
@@ -0,0 +1,15 @@
+from remotespark.utils.log import Log
+
+
+class EventsHandler:
+    def __init__(self):
+        self.logger = Log("EventsHandler")
+
+    def store_event(self, **kwargs):
```

Comment:
The point is that it will matter for other implementations of `EventsHandler` so you need to be able to have an interface that is appropriately generic that it works for everything.